kbs/src/api_server.rs

```diff
@@ -280,9 +283,18 @@ pub(crate) async fn api(
 
                 let claim_str = serde_json::to_string(&claims)?;
 
+                KBS_POLICY_EVALS.inc();
                 // TODO: add policy filter support for other plugins
-                if !core.policy_engine.evaluate(&endpoint, &claim_str).await? {
+                if !core
+                    .policy_engine
+                    .evaluate(&endpoint, &claim_str)
+                    .await
+                    .inspect_err(|_| KBS_POLICY_ERRORS.inc())?
+                {
+                    KBS_POLICY_VIOLATIONS.inc();
                     return Err(Error::PolicyDeny);
+                } else {
+                    KBS_POLICY_APPROVALS.inc();
                 }
```

Comment:
Some personal taste - we even do not need the `else` here

Reply:
done